### PR TITLE
Upgrade PS 7.2 SDK to 7.2.15

### LINKED ISSUE #1003: Upgrade PowerShell 7.2 to SDK 7.2.15
For more information, please see https://github.com/PowerShell/PowerShell/releases/tag/v7.2.15.

### PR DESCRIPTION
**Pull request by andystaples**
### Upgrade PS 7.2 SDK to 7.2.15

resolves #1003

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: PowerShellWorker.Common.props [hunk 1]
```diff
@@ -6,5 +6,6 @@
     <PackageLicenseUrl>https://github.com/Azure/azure-functions-powershell-worker/blob/dev/LICENSE</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/Azure/azure-functions-powershell-worker</RepositoryUrl>
+    <UseRidGraph>Alternate</UseRidGraph>
```
Comment (Francisco-Gamino) [inline PowerShellWorker.Common.props]: Could you please include a comment in the PR description explaining why we need this for `PowerShell 7.2`?

File: tools/helper.psm1 [hunk 2]
```diff
@@ -17,8 +17,8 @@ $DotnetSDKVersionRequirements = @{
     }
 
     '6.0' = @{
-        MinimalPatch = '412'
-        DefaultPatch = '412'
+        MinimalPatch = '415'
```
Comment (Francisco-Gamino) [inline tools/helper.psm1:20]: Could you please add a comment in your commit message that describes this change? E.g., `Upgrade to .NET SDK 6.0.415`?

File: PowerShellWorker.Common.props (hunk 1 above)
Comment (andystaples) [reply PowerShellWorker.Common.props]: This change was actually unintentional - reverted

File: tools/helper.psm1 (hunk 2 above)
Comment (andystaples) [reply tools/helper.psm1:20]: Added to the latest commit message, and I'll ensure it's included in the squash commit message

**Review by Francisco-Gamino — APPROVED** [review]
LGTM

**Pull request merged**